composer.json

```diff
@@ -10,15 +10,18 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9",
-        "illuminate/console": "~5.1|^6.0|^7.0",
-        "illuminate/contracts": "~5.1|^6.0|^7.0",
-        "illuminate/support": "~5.1|^6.0|^7.0",
-        "psy/psysh": "0.7.*|0.8.*|0.9.*",
-        "symfony/var-dumper": "~3.0|~4.0"
+        "php": "^7.2",
+        "illuminate/console": "^6.0|^7.0",
+        "illuminate/contracts": "^6.0|^7.0",
+        "illuminate/support": "^6.0|^7.0",
+        "psy/psysh": "^0.9.0",
```

Comment:
`^0.9` is the same as `^0.9.0` and is more conventional.

Reply:
Updated